eng/pipelines/runtimelab/runtimelab-post-build-steps.yml

```diff
@@ -23,15 +24,17 @@ steps:
       artifactName: CoreCLRProduct__${{ parameters.osGroup }}${{ parameters.osSubgroup }}_${{ parameters.archType }}_${{ parameters.buildConfig }}
       displayName: 'CoreCLR product build'
 
-  - template: /eng/pipelines/common/upload-artifact-step.yml
-    parameters:
-      rootFolder: $(Build.SourcesDirectory)/artifacts/helix
-      includeRootFolder: true
-      archiveType: $(archiveType)
-      archiveExtension: $(archiveExtension)
-      tarCompression: $(tarCompression)
-      artifactName: libraries_test_assets_${{ parameters.osGroup }}_${{ parameters.archType }}_${{ parameters.buildConfig }}
-      displayName: Test Assets
+  # Zip Test Build
+  - ${{ if eq(parameters.uploadTests, 'true') }}:
```

Comment:
Default the parameter to false, and do a boolean comparison, it is usually better to  get precise results, comparing against a string could mess up the build based on casing or other evaluations that azdo does.